azure-pipelines.yml

```diff
@@ -78,6 +78,23 @@ stages:
         npm install tslint --verbose
         npm run lint
       displayName: Run Lint
+  ##### Verify NPM and Yarn are in sync #####
+  - job: SyncPackageManagers
+    displayName: 'Verify NPM & Yarn In-Sync [Local Copy of Target Branch Must Be Up to Date]'
+    pool:
+      vmImage: 'windows-latest'
+    steps:
+    - task: UsePythonVersion@0
```

Comment:
Interesting, out of curiosity why use Python? Familiarity I assume?

Reply:
Yea, familiarity + speed of development time. Writing this in bash sounds... not fun. C# is an option though it's also not really in the repository much so either way it's an introduced new onboarding cost. My hope is someday VSCE will support npm and this script can be eradicated, but since their issues for NPM have been open for a few years now, I don't expect it to happen anytime soon

Reply:
Makes total sense!

@dbaeumer is there someone we can chat with to better understand VSCE expectations / details around yarn / npm integrations?

Reply:
Thanks for linking, I did reach out to Joao. Here are the relevant issues in VSCE's repo https://github.com/microsoft/vscode-vsce/issues/308
https://github.com/microsoft/vscode-vsce/issues/580
https://github.com/microsoft/vscode-vsce/issues/300 
https://github.com/microsoft/vscode-vsce/issues/381

Reply:
@NTaylorMullen the person to talk to is @joaomoreno

Reply:
> My hope is someday VSCE will support these parts of npm

What specific features are you referring to?

Reply:
> What specific features?

Thanks for following up (offline as well a while back.) All of the above are relevant but here are the main 2: https://github.com/microsoft/vscode-vsce/issues/308 and https://github.com/microsoft/vscode-vsce/issues/580